detox/src/devices/DeviceRegistry.js

```diff
@@ -20,8 +20,11 @@ class DeviceRegistry {
   async allocateDevice(getDeviceId) {
     return this._lockfile.exclusively(async () => {
       const deviceId = await safeAsync(getDeviceId);
-      this._toggleDeviceStatus(deviceId, true);
-      return deviceId;
+      const newState = this._toggleDeviceStatus(deviceId, true);
+      return {
```

Comment:
@d4vidi, please fix JSDoc `@returns`

Reply:
@d4vidi could you add samples for two devices connection, please...

Reply:
`Promise<Object>`?

Reply:
@admizh wdym by that?